typeorm: add page (#7474)
tldr-pages/tldr: pages/common/typeorm.md

# typeorm

> > A JavaScript ORM that can run in Node.js, browsers, Cordova, Ionic, React Native, NativeScript, and Electron platforms.

Reviewer: Tiny suggestion
```suggestion
> A JavaScript ORM that can run on Node.js, browser, Cordova, Ionic, React Native, NativeScript, and Electron platforms.
```

> More information: <https://typeorm.io/>.

> - Generate initial TypeORM project structure:

Reviewer: Perhaps this will read better?
```suggestion
- Generate a new initial TypeORM project structure:
```

`typeorm init`

- Create an empty migration file:

`typeorm migration:create --name {{migration_name}}`

> - Create a migration file with the SQL sentences to update the schema:

Reviewer: Never heard of an SQL sentence before. Did you mean `statement` here instead?

`typeorm migration:generate --name {{migration_name}}`

- Run all pending migrations:

`typeorm migration:run`

> - Create a new entity file into a specific directory:

Reviewer: Perhaps this will read better?
```suggestion
- Create a new entity file in a specific directory:
```

`typeorm entity:create --name {{entity}} --dir {{path/to/directory}}`

> - Display the SQL sentences to be executed by `typeorm schema:sync` on the default connection:

Reviewer: Again with the SQL sentences. Is this some kind of tool-specific technical language? 'cause I certainly haven't heard of it elsewhere.

`typeorm schema:log`

> - Execute a specific SQL sentence on the default connection:

Reviewer: Same here.
